app/controllers/concerns/body_class_concern.rb

```diff
@@ -0,0 +1,32 @@
+module BodyClassConcern
+  extend ActiveSupport::Concern
+  extend Mandate::Memoize
+
+  memoize
+  def body_class
+    classes = []
+    classes << "devise" if devise_controller?
+    classes << "namespace-#{namespace_name}"
+    classes << "controller-#{controller_name || 'none'}"
+    classes << "action-#{action_name}"
+    classes << "theme-light"
+    user_signed_in? ? classes << "user-signed_in" : classes << "user-signed_out"
```

Comment:
I assume rubocop doesn't let you do:

```suggestion
    classes << user_signed_in? ? "user-signed_in" : "user-signed_out"
```

Reply:
Seems to work with brackets! 🎉